assets/rnadeseq_report.Rmd

```diff
@@ -1835,62 +2048,68 @@ for (file in allgenes_files){
                         )
                     )
                 )
             )
-    log2FoldChange_min <- min(DE_genes$log2FoldChange)
-    log2FoldChange_max <- max(DE_genes$log2FoldChange)
-    log2FoldChange_quotient <- length(seq(log2FoldChange_min, log2FoldChange_max, 10))
 
-    pg <- ggplot(
-            DE_genes,
-            aes(
-                x=log2FoldChange,
+    DE_genes$color_code <- as.factor(DE_genes$color_code)
+    color_map <- c(
+        setNames("#d03a33", label_DE),
+        setNames("#458a34", label_logFC),
+        setNames("#444444", label_NS),
+        setNames("#465adb", label_padj)
+    )
+
+    pg <- ggplot(DE_genes,
+            aes(x=log2FoldChange,
                 y=logpval,
+                colour=color_code,
                 text=paste("Gene: ", gene_name,
                             "<br>", "Log2FC: ", formatC(log2FoldChange, digits=2),
-                            "<br>", "-Log10padj: ", formatC(truelogpval, digits=2, format='f')))) +
-            geom_point(
-            alpha = 0.5, aes(colour = ifelse(color == "red", paste0("DE:", "p_adj<", params$adj_pval_threshold, " & Log2FC >|", params$logFC_threshold, "|"),
-                ifelse(color == "blue", paste0("p_adj<", params$adj_pval_threshold),
-                ifelse(color == "green", paste0("log2FC>|", params$logFC_threshold, "|"),
-                    ifelse(color == "black", "NS", "")
-                )
-                )
-            )),
-            na.rm = TRUE
-            ) +
+                            "<br>padj: ", formatC(padj, digits = 2),
+                            "<br>", "-Log10padj: ", formatC(truelogpval, digits=2, format='f'))
+                            )) +
+            geom_point(alpha = 0.5,
+                        na.rm = TRUE) +
             geom_hline(yintercept = -log10(params$adj_pval_threshold), linewidth = 0.2, color = "grey") +
             geom_vline(xintercept = -1, linewidth = 0.2, color = "grey") +
             geom_vline(xintercept = 1, linewidth = 0.2, color = "grey") +
-            scale_color_manual(name = "", values = c("red", "green", "black", "blue")) +
+            scale_color_manual(name = "", values=color_map) +
             scale_y_continuous(limits = c(-1, max(-log10(DE_genes[['padj']]), na.rm=TRUE) + 5)) +
             ylab("-log 10 padj") + xlab("Log2 Fold Change") +
             theme(panel.spacing = unit(ifelse(length(allgenes_files)>2, 0.1, 0.6), "lines")) +
             scale_x_continuous(breaks = seq(-200, 200, ifelse(log2FoldChange_quotient<3, 1, 5))) +
-            ggtitle(
-                paste('Comparison:', current_contrast_name),
-                subtitle=paste0("DE results; total=", thousandFormat(nrow(DE_genes)), ", DE=", thousandFormat(numDE)))
+            ggtitle(paste('Comparison:', current_contrast_name),
+                    subtitle=paste0("DE results; total=",
```

Comment:
Is total indeed so small? Isnt that maybe some filtered table, but I would expect here the number of all genes in the GTF?
Maybe use `num_all` as in the `Summary` section? That appears to be the real total number of genes?!

Reply:
Just wanted to ask whether thats updated because I dont see an update (but probably miss it in the code)

Reply:
I had updated it, but disappeared. Will add it again then